pilot/lib/pilot/build_manager.rb

```diff
@@ -61,7 +61,8 @@ def distribute(options, build = nil)
           return
         end
 
-        UI.message("Distributing build #{build.train_version}(#{build.build_version}) from #{build.testing_status} -> External")
+        type = options[:distribute_external] ? 'External' : 'Internal'
+        UI.message("Distributing build #{build.train_version}(#{build.build_version}) from #{build.testing_status} -> #{type}")
```

Comment:
I wonder if we should also change the text below:

```ruby
       UI.message("Successfully distributed build to beta testers 🚀")
```
and replace "beta" with "External" or "Internal". While we are at it, maybe make it a `UI.success`